rust/core-lib/src/syntax.rs

```diff
@@ -120,3 +121,44 @@ impl LanguageDefinition {
         }
     }
 }
+
+// for testing
+#[cfg(test)]
+pub fn language_for_name() {}
+
+#[cfg(test)]
```

Comment:
the general way to write tests is to add a 'test' module; and then each test case is marked with `#[test]`. So this would look like,

```rust
#[cfg(test)]
mod tests {
    use super::*;

    #[test]
    fn my_function() {
        assert_eq!(String::from("one"), "one".to_string());
    }
}
```

`#[cfg(test)]` indicates conditional compliation; this section of code will only be compiled if we're compiling for testing. `#[test]` is a special procedural macro; it says that this function should be turned into a test case, and run when tests are run.

Reply:
Ooh.. that'll be done.